### PR TITLE
docs: document how to run workspace-proxy as a system service

### LINKED ISSUE #12783: Doc: Documentation on how to run a workspace proxy via `systemctl`
Current docs describe how to setup a Workspace Proxy via the CLI:

- https://coder.com/docs/v2/latest/admin/workspace-proxies

Some docs should be added like:

```bash
# Edit this file /etc/coder.d/coder-workspace-proxy.env
sudo systemctl enable --now coder-workspace-proxy
```

PR #12810

### PR DESCRIPTION
**Pull request by michaelbrewer**
closes #12783

FYI @kylecarbs

### CODE REVIEW

**Review by kylecarbs — APPROVED** [review]

**Review by f0ssel — APPROVED** [review]
This is awesome, thanks so much!

File: docs/admin/workspace-proxies.md [hunk 1]
```diff
@@ -147,6 +147,27 @@ and up the deployment's replicas.
 coder wsproxy server
 ```
 
+### Running as a system service
+
+If you've installed Coder via a [system package](../install/index.md) Coder, you
```
Comment (matifali) [inline docs/admin/workspace-proxies.md:152]:
```suggestion
If you've installed Coder via a [system package](../install/index.md), you
```

Comment (matifali) [reply docs/admin/workspace-proxies.md:152]: duplication

Comment (michaelbrewer) [reply docs/admin/workspace-proxies.md]: copy and pasta. let me fix the source too..

**Comment by michaelbrewer** [comment]
@matifali - i am not sure how to get this to build right. `.devcontainer` fails, and `nix-shell` fails to..

**Comment by matifali** [comment]
@michaelbrewer I will run the `make fmt` for you. And `nix develop` should work. I just did it using nix. I will check `.devcontainer`.

**Pull request merged**